android/app/src/main/java/org/cryptimeleon/incentive/app/domain/usecase/ResetAppUseCase.kt

```diff
@@ -0,0 +1,17 @@
+package org.cryptimeleon.incentive.app.domain.usecase
+
+import org.cryptimeleon.incentive.app.domain.IBasketRepository
+import org.cryptimeleon.incentive.app.domain.ICryptoRepository
+import org.cryptimeleon.incentive.app.domain.IPromotionRepository
+
+class ResetAppUseCase(
+    private val cryptoRepository: ICryptoRepository,
+    private val basketRepository: IBasketRepository,
+    private val promotionRepository: IPromotionRepository
+) {
+    suspend operator fun invoke() {
```

Comment:
does this also make the onboarding screen reappear?

Reply:
No, it starts with the loading screen again to load pp, keys, promotions, etc